Formula/nettle.rb

```diff
@@ -26,9 +26,13 @@ def install
                                           "get_pc_thunk|(_*chkstk_darwin)"
     end
 
+    # Disable assembly as a temporary measure
```

Comment:
Please file an upstream issue if none exist and link it here

Reply:
Can build arm64 native (without disabling assembly) with the fix in 5a5b4f4